src/main/java/hudson/plugins/octopusdeploy/AbstractOctopusDeployRecorderPostBuildStep.java

```diff
@@ -298,7 +296,7 @@ public OctopusApi getApi() {
         checkState(StringUtils.isNotBlank(apiKey), String.format(OctoConstants.Errors.INPUT_CANNOT_BE_BLANK_MESSAGE_FORMAT, "API Key"));
 
         commands.add(OctoConstants.Commands.Arguments.PROJECT_NAME);
-        commands.add(project);
+        commands.add(envInjector.injectEnvironmentVariableValues(project));
```

Comment:
This wasn't getting evaluated previously

Reply:
I've kept the `Space` selection a dropdown, because the it displays the space name while the backing field is the ID. This isn't possible with combobox.

Reply:
👍